src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,120 @@
+
+When the user interacts with the secondary link, it will perform the same action as pressing the button to activate Exit this Page.
+
+[EXAMPLE OF SECONDARY LINK]
+
+The component will still work if the secondary link is not added, but it'll be less accessible overall.
+
+## Enhancement features
+
+These features are included in the component and should work automatically, as long as the user has enabled Javascript their browser. You do not need to do anything to make these work.
+
+### Loading overlay
+
+The 'loading overlay' will appear immediately when the user activates Exit this page.
+
+The overlay helps users with slow connections clear their screen until the browser loads the next page.
+
+### Showing progress for 'Shift key' presses
+
+To show the user the number of times they’ve pressed the shift button, there’s a series of 3 progress dots under the button, provided as a visual enhancement. Each time the shift key is pressed, one of the dots will be filled.
+
+The user has 5 seconds to press the shift key 3 times. After that, the sequence and progress dots will reset.
+
+## Accessibility features and considerations
+
+In addition to the 'Exit this page' button itself, users have 2 other options to activate the component. We've tested and iterated both of these options to be accessible, reliable and as discreet as possible for users.
+
+The keyboard option, where the user can press shift 3 times, gives users a discreet way to activate the component, particularly keyboard-only users. We considered options that used a single key press (such as the 'Esc' key), but we could not be confident that this would not interfere with other user actions.
+
+The 'secondary link' option is a hidden link in the tab order of the page. This gives users of assistive technology and easier way to reliably activate the component.
+
+To help users of speech recognition software activate the 'secondary link' more discreetly, we considered  renaming the 'Exit this page' link into something else that hid its real purpose. 
+
+However, we decided this was more likely to confuse users. We're also confident about the other ways users can use speech recognition software to activate the 'secondary link' without saying the text out loud, such choosing the link as an item number on a list.
+
+## Research on this component
+
+The design of this component is based on research with people with a lived experience of domestic abuse and people with accessibility needs, and in consultation with the Ministry of Justice, Department for Work and Pensions and the Scottish Government.
+
+[Tell us if your service uses this component (opens in a new tab)](https://www.smartsurvey.co.uk/). This will help us improve this component to better meet the needs of the services that use it.
```

Comment:
Need to add link to the survey once we create it.
```suggestion
[[ADD LINK] Tell us if your service uses this component (opens in a new tab)](https://www.smartsurvey.co.uk/). This will help us improve this component to better meet the needs of the services that use it.
```